io.github.mahmoudbahaa.outlook_for_linux.yml

```diff
@@ -0,0 +1,50 @@
+app-id: io.github.mahmoudbahaa.outlook_for_linux
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: outlook-for-linux
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --filesystem=home
+  - --filesystem=xdg-run/dconf
+  - --filesystem=~/.config/dconf:ro
+  - --env=DCONF_USER_CONFIG_DIR=.config/dconf
+  - --talk-name=ca.desrt.dconf
+rename-desktop-file: outlook-for-linux.desktop
+rename-icon: outlook-for-linux
+modules:
+  - name: outlook-for-linux
+    buildsystem: simple
+
+    sources:
+      - type: file
+        url: https://github.com/mahmoudbahaa/outlook-for-linux/releases/download/v1.3.13-outlook/outlook-for-linux_1.3.13_amd64.deb
```

Comment:
does this include a vulneraable Electron runtime?

Also why no aarch64?

Reply:
no  electron 27 https://github.com/mahmoudbahaa/outlook-for-linux/blob/develop-outlook/package.json 
added aarch64

Reply:
sorry it was 24.6.7 in the latest release i made a build from the pre-release with updated electron 27.0.0 and updated urls in manifest

Reply:
24.6.7 is vulnerable.